### PR TITLE
Facilitator slide

### PR DESCRIPTION
**Pull request by gewfy**

### CODE REVIEW

File: functions/src/api/temples/index.spec.ts [hunk 1]
```diff
@@ -58,6 +61,14 @@ const mockServer = createMockServer(
...
 );
 
+beforeEach(async () => {
+  await Promise.all(
+    temples.map(temple =>
+      firestore().collection('temples').doc(temple.id).set(temple),
+    ),
+  );
+});
+
```
Comment (gewfy) [inline functions/src/api/temples/index.spec.ts:71]: Like it! Merge all the things!

**Comment by Nipher** [comment]
The call doesn't seem to work anymore on my ios emulator for some reason, neither the disable/enable camera/mic...

Will build to my android now to see what happens

**Comment by Nipher** [comment]
Rebuilt the app now and it works 🎉

**Comment by Nipher** [comment]
@gewfy @swemail how do you feel about [481719c](https://github.com/29ki/29k/pull/274/commits/481719c1160f76e705a3e3d6709d30759e1b6e04)?

My intention there was to provide a way for any component/content to let the selector know who's on spotlight, so we can adapt/format the list of participants with that in mind, instead of doing that on the view layer.

**Review by swemail — APPROVED** [review]

**Pull request merged**